frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -0,0 +1,442 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { Link } from 'react-router-dom';
+import { ClusterServiceVersionLogo, iconFor } from './index';
+import {
+  ActionGroup,
+  Alert,
+  Bullseye,
+  Button,
+  Card,
+  CardBody,
+  Spinner,
+} from '@patternfly/react-core';
+import {
+  Firehose,
+  FirehoseResult,
+  ResourceLink,
+  ResourceStatus,
+  resourcePathFromModel,
+} from '@console/internal/components/utils';
+import {
+  groupVersionFor,
+  k8sPatch,
+  referenceForModel,
+  referenceForGroupVersionKind,
+} from '@console/internal/module/k8s';
+import { errorModal } from '@console/internal/components/modals';
+import {
+  ClusterServiceVersionModel,
+  InstallPlanModel,
+  PackageManifestModel,
+  SubscriptionModel,
+} from '../models';
+import {
+  ClusterServiceVersionKind,
+  SubscriptionKind,
+  InstallPlanKind,
+  PackageManifestKind,
+} from '../types';
+import { StatusIconAndText } from '@console/shared';
+import {
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+} from '@console/shared/src/components/status/icons';
+import { InstallPlanPreview } from './install-plan';
+
+const getCRDFromObject = (obj: ClusterServiceVersionKind | InstallPlanKind | SubscriptionKind) => {
+  return obj?.metadata?.annotations?.['operatorframework.io/initialization-resource'];
+};
+
+const ViewInstalledOperatorsButton: React.FC<InstallPageMessageButtonProps> = ({
+  namespace,
+  className,
+}) => (
+  <Link to={resourcePathFromModel(ClusterServiceVersionModel, null, namespace)}>
+    <Button className={className} data-test="view-installed-operators-btn" variant="link">
```

Comment:
If this is styled as a link and inside a `Link` component, is `Button` needed at all?

Reply:
Without the button, the styling gets messed up and unaligned.

Reply:
> Without the button, the styling gets messed up and unaligned.

How so? We should try to fix the CSS instead of adding an unnecessary `button` element.